frontend/components/login/LoginProviders.tsx

```diff
@@ -34,17 +70,29 @@ export default function LoginProviders({providers,login_info_url}:LoginProviders
                   '&:hover': {
                     opacity: 1,
                     backgroundColor:'background.default'
-                  }
+                  },
+                  position:'relative'
                 }}
               >
-                <ListItemText
-                  primary={provider.name}
-                  secondary={provider?.html ?
-                    <span
-                      dangerouslySetInnerHTML={{__html: provider.html}}
+                {
+                  provider?.accessType==='INVITE_ONLY' ?
+                    <Chip
+                      size='small'
+                      label={provider?.accessType}
```

Comment:
This is the right line, as written before I would prefer to replace `_` with spaces, for example like this:

```suggestion
                      label={provider?.accessType?.replace('_', ' ')}
```